src/c4/ifpstream.cc

```diff
@@ -0,0 +1,69 @@
+//----------------------------------*-C++-*-----------------------------------//
+/*!
+ * \file   c4/ifpstream.cc
+ * \author Mathew Cleveland
+ * \date   Feb. 2020
+ * \brief  Define methods of class ifpstream
+ * \note   Copyright (C) 2018-2020 Triad National Security, LLC.
+ *         All rights reserved. */
+//----------------------------------------------------------------------------//
+
+#include "ifpstream.hh"
+#include "C4_Functions.hh"
+
+namespace rtt_c4 {
+using namespace std;
+
+//----------------------------------------------------------------------------//
+/*! Create an ifpstream to for a parallel file read.
```

Comment:
Please use `\brief` for the one-line quick summary (ditto for all the new code).  This is only needed if the documentation block is longer than a single line

Reply:
Fixed.